calicoctl.py

```diff
@@ -793,16 +785,28 @@ def bgppeer_add(ip, version):
     :param version: v4 or v6
     :return: None
     """
+    address = check_ip_version(ip, version, IPAddress)
+    client.add_bgp_peer(version, address)
+
+
+def check_ip_version(ip, version, type):
```

Comment:
Please rename `type` to `cls`, which is the Python convention for that sort of thing.  `type` is a built-in function (and `class` is a reserved word), so it's easy to misread this code and it makes it less portable.